.travis.yml

```diff
@@ -7,13 +7,17 @@ python:
     - 3.6
     - nightly
 
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
```

Comment:
Not that it's really important right now, but is there a difference for using a different distro of ubuntu for the tests? I think i'd prefer that either all tests are done on the same distro or that we redo each test for each distro. You can set the environment of all the tests to Xenial if you prefer, however

Reply:
It is just because the default did not found a package for python3.7. But as it is working like other python versions maybe we can remove it. I just wanted to use python3.7 because it is the one I used while developing.